addons/website_hr_recruitment/models/hr_applicant.py

```diff
@@ -25,3 +25,14 @@ def website_form_input_filter(self, request, values):
             if stage:
                 values['stage_id'] = stage.id
         return values
+
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
+        groups = super()._notify_get_recipients_groups(message, model_description, msg_vals=msg_vals)
+        local_msg_vals = dict(msg_vals or {})
+        customer_group = next(group for group in groups if group[0] == 'customer')
+        customer_group[2]['active'] = True
+        customer_group[2]['has_button_access'] = True
+        access_opt = customer_group[2].setdefault('button_access', {'title': _('Job Description')})
+        access_opt['url'] = self._notify_get_action_link('controller', controller=self.job_id.website_url, **local_msg_vals)
```

Comment:
Can we update the dict in one statement instead of updating the keys one by one?